docs/navbar.js

```diff
@@ -0,0 +1,10 @@
+export {loadNavbar};
+
+/**
+ * Loads the navbar and calls the callback upon load.
+ *
+ * @param {Function} callback the function called upon load
+ */
+function loadNavbar(callback) {
```

Comment:
it looks like the callbacks for this function are all of the same form. Could we just pass the string in as an argument and have this construct the callback?

Reply:
In #218, the child PR, this is not the case that's why I set it up like this. Fixed in this PR.